modules/hcp-get-ocp-channel.adoc

```diff
@@ -2,134 +2,74 @@
 //
 // * hosted_control_planes/hcp-updating.adoc
 
-:_mod-docs-content-type: CONCEPT
+:_mod-docs-content-type: PROCEDURE
 [id="hcp-get-ocp-channel_{context}"]
-= Setting channels in a hosted cluster
+= Updating your hosted cluster
 
-You can see available updates in the `HostedCluster.Status` field of the `HostedCluster` custom resource (CR).
+You can update your hosted cluster by setting a channel that matches with your desired channel semantics.
 
-The available updates are not fetched from the Cluster Version Operator (CVO) of a hosted cluster. The list of the available updates can be different from the available updates from the following fields of the `HostedCluster` custom resource (CR):
+The initial `HostedCluster` resource does not have any information in the `status.version.availableUpdates` and `status.version.conditionalUpdates` fields. After you set the `spec.release` field to the {product-title} release version, the HyperShift Operator updates the `status.version` field with the available and conditional updates.
 
-* `status.version.availableUpdates`
-* `status.version.conditionalUpdates`
+.Procedure
 
-The initial `HostedCluster` CR does not have any information in the `status.version.availableUpdates` and `status.version.conditionalUpdates` fields. After you set the `spec.channel` field to the stable {product-title} release version, the HyperShift Operator reconciles the `HostedCluster` CR and updates the `status.version` field with the available and conditional updates.
+. Check recommended {product-title} channels by running the following command:
++
+[source,terminal]
+----
+$ oc -n <hosted_cluster_namespace> get -o jsonpath='{.status.version.desired.channels}' hostedcluster <hosted_cluster_name>
+----
 
-See the following example of the `HostedCluster` CR that contains the channel configuration:
+. Set a channel in your hosted cluster that matches with your desired channel semantics. For more information about channels and releases, see "Understanding update channels and releases". Run the following command:
++
+[source,terminal]
+----
+$ oc patch -n <hosted_cluster_namespace> hostedcluster <hosted_cluster_name> --type=merge -p '{"spec":{"channel":"<channel>"}}' <1>
+----
+<1> Replace `<channel>` with the {product-title} release version that you specified in `spec.release`. For example, if you set the `spec.release` to `ocp-release:4.18.4-multi`, you can set `spec.channel` to candidate-4.18, fast-4.19, candidate-4.20, and so on. For more information about channels, see "Understanding update channels and releases".
```

Comment:
While you certainly can set `candidate-4.18`, etc., [they can be unsupported, early-access releases][1].  I'd rather use `fast-{product-version}` and `stable-{product-version}` as examples here, to reduce the chances that someone reads this section, but does not read the referenced _Understanding update channels and releases_ section, and sets `candidate-*` without realizing that channel's semantics.

[1]: https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/updating_clusters/understanding-openshift-updates-1#candidate-version-channel_understanding-update-channels-releases